src/Compilers/CSharp/Test/Emit3/Semantics/ExtensionTests.cs

```diff
@@ -17834,4 +17832,366 @@ static class E
+            //         public int P1 => 4;
+            Diagnostic(ErrorCode.ERR_MemberAlreadyExists, "4").WithArguments("get_P1", "Extensions").WithLocation(10, 26)
+            );
+    }
+
+    [Fact]
+    public void SignatureConflict_04()
+    {
+        var src = """
+static class Extensions
+{
+    extension(object receiver)
+    {
+        static public int P1 => 1;
+    }
+
+    extension(object receiver)
+    {
+        static public int P1 => 4;
+    }
+}
+""";
+        var comp = CreateCompilation(src);
+
+        comp.VerifyDiagnostics(
+            // (10,33): error CS0111: Type 'Extensions' already defines a member called 'get_P1' with the same parameter types
+            //         static public int P1 => 4;
+            Diagnostic(ErrorCode.ERR_MemberAlreadyExists, "4").WithArguments("get_P1", "Extensions").WithLocation(10, 33)
+            );
+    }
+
+    [Fact]
+    public void SignatureConflict_05()
+    {
+        var src = """
+static class Extensions
+{
+    extension(object receiver)
+    {
+        public int P1 => 1;
+        public int get_P2() => 2;
+    }
+
+    extension(object receiver)
+    {
+        public int get_P1() => 3;
+        public int P2 => 4;
+    }
+}
+""";
+        var comp = CreateCompilation(src);
+
+        comp.VerifyDiagnostics(
+            // (11,20): error CS0111: Type 'Extensions' already defines a member called 'get_P1' with the same parameter types
+            //         public int get_P1() => 3;
+            Diagnostic(ErrorCode.ERR_MemberAlreadyExists, "get_P1").WithArguments("get_P1", "Extensions").WithLocation(11, 20),
+            // (12,26): error CS0111: Type 'Extensions' already defines a member called 'get_P2' with the same parameter types
+            //         public int P2 => 4;
+            Diagnostic(ErrorCode.ERR_MemberAlreadyExists, "4").WithArguments("get_P2", "Extensions").WithLocation(12, 26)
+            );
+    }
+
+    [Fact]
+    public void SignatureConflict_06()
+    {
+        var src = """
+static class Extensions
+{
+    extension(object receiver)
+    {
+        public int P1 => 1;
+    }
+
+    extension(object receiver)
+    {
+        public int P1 {set{}}
+    }
+}
+""";
+        var comp = CreateCompilation(src);
+
+        // PROTOTYPE: Should be complain about a conflict like this?
+        CompileAndVerify(comp).VerifyDiagnostics();
+    }
+
+    [Fact]
+    public void SignatureConflict_07()
+    {
+        var src = """
+static class Extensions
+{
+    extension(object receiver)
+    {
+        public void M(){}
+    }
+
+    extension(__arglist)
+    {
+        public void M(object x){}
+    }
+}
+""";
+        var comp = CreateCompilation(src);
+
+        comp.VerifyDiagnostics(
+            // (8,15): error CS1669: __arglist is not valid in this context
+            //     extension(__arglist)
+            Diagnostic(ErrorCode.ERR_IllegalVarArgs, "__arglist").WithLocation(8, 15)
+            );
+    }
```

Comment:
Consider testing:
- indexers
- conflict where the difference is in the receiver parameter name

Reply:
> indexers

I don't find indexers very interesting. I covered properties only to make it obvious how the errors would look like, i.e. errors will be about accessors.

> conflict where the difference is in the receiver parameter name

Since I am not writing new code that checks for collisions (simply making sure it is called for implementation methods), I do not find it interesting to cover in context of this PR, since names are not part of the signature given the current language rules. 

For the sake of expediency, I am not planning to add these tests in this PR, but I mentioned scenarios in the Test Plan issue.